org.lflang.tests/src/org/lflang/tests/runtime/CTest.java

```diff
@@ -59,6 +59,11 @@ protected boolean supportsFederatedExecution() {
         return true;
     }
 
+    @Override
+    protected boolean supportsDockerOption() {
```

Comment:
We could use this opportunity to create an issue for what remains to be done for Python. I have created a new project called [Docker Support](https://github.com/lf-lang/lingua-franca/projects/11) to track the progress.

Reply:
I added docker tests for non-federated python programs